### PR TITLE
records: sync modifications in database and ElasticSearch

### PR DESCRIPTION
**Pull request by jmartinm**
* Adds models_committed receiver that sends to ElasticSearch the changes
  made in the RecordsMetadata table.

Signed-off-by: Javier Martin Montull <javier.martin.montull@cern.ch>

### CODE REVIEW

**Comment by jmartinm** [comment]
Ready for review

File: inspirehep/modules/migrator/cli.py [hunk 1]
```diff
@@ -54,6 +57,8 @@
 def migrator():
     """Command related to migrating INSPIRE data."""
     logging.basicConfig()
+    # Disable auto-indexing receiver in migration tasks
+    models_committed.disconnect(receive_after_model_commit)
```
Comment (jmartinm) [inline inspirehep/modules/migrator/cli.py]: Now that I think about it, shall I connect it again? Or the migrator is a one-of CLI command and future requests will have the signal connected?

Comment (jacquerie) [inline inspirehep/modules/migrator/cli.py]: I would expect this to leave it disconnected, so I'm not sure if this is correct, but honestly I have no idea...

Comment (jmartinm) [inline inspirehep/modules/migrator/cli.py]: To be on the safe side I am reconnecting the signal https://github.com/inspirehep/inspire-next/pull/1775/files#diff-7e8feab1a9e8ec565a17aca30402e654R66

If Travis passes it is ready to be merged.

**Pull request merged**